src/regexps.ts

```diff
@@ -28,12 +28,12 @@ export const anyLine = XRegExp('^ (?<line> .* ) $', 'nx');
 
 
 // Re-builds the 🍂 directive based on a different leading character
-export function redoLeafDirective(char) {
+export function redoLeafDirective(char: string): any {
```

Comment:
I added `any`s here and there just to get it working. There could be a second pass at defining meaningful types.